### ISSUE TITLE
Duplicate scene ids for scenes with same geometry / datetime but different receiving stations

### ISSUE THREAD

**Issue by jonas-eberle** [stactools-packages/sentinel2#5]
Hi all, 

with a recent commit (https://github.com/stactools-packages/sentinel2/commit/f4aa3572e7437f11926c11d7d0e9b1d281241e95) the scene_id is now changed with the removal of the processing baseline number and the product discriminator. 

Especially the removal of the product discriminator seems to be critical as there are scenes available, which come from two receiving stations (with overlapping areas). See following example. 
  
![image](https://user-images.githubusercontent.com/2070954/124486920-60063680-ddae-11eb-9491-ddf35cc43488.png)

Scene IDs: 
- S2A_MSIL2A_20200210T073031_N0214_R049_T37MEU_20200210T130912 (L1C data received from EPAE - Madrid)
- S2A_MSIL2A_20200210T073031_N0214_R049_T37MEU_20200210T100652 (L1C data received from SGS - Svalbard)

Using stactools-sentinel2 both scenes have the same scene id (according to the recent commit) but as they have been received by two different stations the content of the scene is different. 

How do you handle this? From my point of view I would at least keep both scenes. One (maybe not easy) solution might be to merge both scenes on a raster level because any user needs to take care of this otherwise in the data analysis part (but this means data needs to be changed and metadata needs to be recalculated). 

Best
Jonas

**Comment by lossyrob** [comment]
Thanks for raising this issue! It was my understanding that the end date was a processing number that was particular to the date the product was processed. This seems like a misunderstanding - so the product discriminator is actually used to distinguish between scenes that were captured on the same day between two receiving stations?

Seems like we'll have to make a change to put the product discriminator back into the ID. I think dropping the baseline number (and the .SAFE extension) is still a good change; we saw reprocessed products between versions of Sen2Core end up with different Item IDs based on reprocessing without it, which is undesirable.

**Comment by jonas-eberle** [comment]
I just looked at the definition of the product discriminator: 
`is used to distinguish between different end user products from the same datatake. Depending on the instance, the time in this field can be earlier or slightly later than the datatake sensing time.`

I just discovered this by inserting several thousand scenes into the STAC API catalogue and getting some "duplicate scene id" error messages. 

Dropping the baseline number and .SAFE extension is fine for me.
